oaknational/Oak-Web-Application
# Feat/lesq 873/legacy downloads use new api [LESQ-873]

## Description

Music year: 2003

- use the new downloads API url for legacy downloads and check-files requests
- refactor curriculum download URL to use env var

## How to test
I've tried to test from a range of subjects/keystages but the more coverage the better

1. Go to https://deploy-preview-2488--oak-web-application.netlify.thenational.academy
2. Click on legacy lessons and download resources
3. requests should be to `https://downloads-api.thenational.academy` and _not_ `https://api.thenational.academy`
4. worth testing [curriculum downloads ](https://deploy-preview-2488--oak-web-application.netlify.thenational.academy/teachers/curriculum/previous-downloads?subject=english-grammar&keystage=ks2)as well, as the URL was refactored


## ACs
- [ ] resources from legacy content download succesfully
- [ ] resources from current content download successfully
